topKFrequentElements.js

```diff
@@ -0,0 +1,36 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/top-k-frequent-elements/
+
+const topKFrequent = (nums, k) => {
+  const numsOccurences = {};
+  for (let i = 0; i < nums.length; i += 1) {
+    if (!numsOccurences[nums[i]]) {
+      numsOccurences[nums[i]] = 1;
+    } else {
+      numsOccurences[nums[i]] += 1;
+    }
+  }
+  const sortedOccurances = [];
+  const numsOccurencesArr = Object.entries(numsOccurences);
+  for (let i = 0; i < numsOccurencesArr.length; i += 1) {
+    const element = numsOccurencesArr[i];
+    const occurence = element[1];
+    const num = +element[0];
```

Comment:
is this a typo ?

Reply:
+ converts string to number. 
Keys in object are strings, I convert them to numbers.